### PR TITLE
Fix travis errors #263, Fix indentation, coding style

### PR DESCRIPTION
**Pull request by fragm3**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [x] Bug fix
- [ ] Feature implementation
- [ ] Doc updates

* **What changes have you introduced?**
 - Fix falling travis error, make travis pass #263
 - Fix indentation
 - Follow a common code style/preference
 - Add enzyme

### CODE REVIEW

**Comment by fragm3** [comment]
@AakashMallik I am having the following problems:

1. The travis is failing due to window.require not available
Error:
Screenshot 2019-04-24 at 2 35 26 PM

I tried looking for solution, [Link](https://github.com/electron/electron/issues/7300). We can either eject from CRA to webpack or use create-react-app-rewired. I would prefer webpack.

2. The indentation of components sent as props is not correct. For instance see `Oscilloscope.js`

**Comment by fragm3** [comment]
@AakashMallik I have added, 
"react/jsx-key": 0,
"react/prop-types": 0,
"no-undef": 0, 
for disabling specific checks for linting. Will enable it on a later stage.

**Comment by AakashMallik** [comment]
@fragm3 Double quotes issue fixed?

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "antd": "^3.16.2",
+    "babel-eslint": "^10.0.1",
```
Comment (AakashMallik) [inline package.json]: please check. Shouldn't this be a dev dependency?

Comment (fragm3) [reply package.json]: @AakashMallik Good catch, made the changes.

**Comment by AakashMallik** [comment]
@mariobehling @CloudyPadmal This is good to get merged. We would start with MUI and end to end oscilloscope implementation as soon as this gets merged.

**Comment by AakashMallik** [comment]
>     * **Please check if the PR fulfills these requirements**
> 
> 
>     * [x]  The commit message follows our guidelines
> 
>     * [x]  Tests for the changes have been added (for bug fixes / features)
> 
>     * [ ]  Docs have been added / updated (for bug fixes / features)
> 
> 
>     * **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
> 
> 
>     * [x]  Bug fix
> 
>     * [ ]  Feature implementation
> 
>     * [ ]  Doc updates
> 
> 
>     * **What changes have you introduced?**
> 
> 
>     * Fix falling travis error, make travis pass
> 
>     * Fix indentation
> 
>     * Follow a common code style/preference
> 
>     * Add enzyme

Add the issue numbers that should get closed automatically... Rest looks perfect to me.

**Comment by fragm3** [comment]
@AakashMallik done

**Comment by CloudyPadmal** [comment]
Hello all, Could we also setup codacy to test code quality? Let's merge this PR and move forward.

**Comment by AakashMallik** [comment]
> Hello all, Could we also setup codacy to test code quality? Let's merge this PR and move forward.

Yes, we would setup dev tools and code quality monitoring as soon as we have completed end to end implementation of the oscilloscope. From there @fragm3 will diverge in setting up dev tools and improving UI, while I'll diverge to optimizing connections and tweaking the py script.

**Pull request merged**